lib/scavenger_hunt/app/models/scavenger_hunt/game.rb

```diff
@@ -33,7 +33,9 @@ def total_time
   def create_clues
     representations = location.representations_by_metum("Scavenger Hunt: Clue")
     representations.each do |representation|
-      clues.create!(game: self, representation: representation)
+      if representation.status == "approved"
```

Comment:
Try `status.approved?` here - status is an enum, so if you don't hard-code these string comparisons we'll catch errors if the names of the statuses ever change!

Reply:
This makes total sense! I tried to make the update and I'm getting the following error. Any ideas? `undefined method `approved?' for "not_approved":String`

Reply:
@flipsasser Any thoughts on the error I mentioned above? (I forgot to tag you, so I imagine you didn't get notified.)